locale/ne.js

```diff
@@ -112,11 +112,11 @@
             yy : '%d बर्ष'
         },
         week : {
-            dow : 1, // Monday is the first day of the week.
-            doy : 7  // The week that contains Jan 1st is the first week of the year.
+            dow : 1, // Sunday is the first day of the week in Nepal
+            doy : 1 // The week that contains Jan 1st is the first week of the year.
```

Comment:
I cannot remember what this was supposed to represent(and why it has been changed). It's been a while.